.github/workflows/rock_branching.yml

```diff
@@ -0,0 +1,50 @@
+name: Run Rock Release Branching
+
+on:
+  workflow_call:
+    inputs:
+      release_branch:
+        type: string
+        default: release/therock-7.x
+        
+  workflow_dispatch:
+    inputs:
+      release_branch:
```

Comment:
workflow_call and workflow_dispatch params should match, have the same description for the same thing and need safe default values or fail graciously when wrong/none values are given